app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,43 @@
+    <string name="app_list">Applications</string>
+    <string name="installed">Installé</string>
+    <string name="app_updates">Mises à jour</string>
+    <string name="install">Installer</string>
+    <string name="uninstall">Désinstaller</string>
+    <string name="update">Mettre à jour</string>
+    <string name="open">Ouvrir</string>
+    <string name="unknown">\???</string>
+    <string name="swipe_refresh">Balayez vers le bas pour rafraîchir</string>
+    <string name="no_apps_installed">Aucune application installée</string>
+    <string name="up_to_date">Toutes les applications sont à jour</string>
+    <string name="user_action_required">Action de l\'utilisateur requise</string>
+    <string name="cant_find_app">Désolé, nous n\'avons pas trouvé cette application</string>
+    <string name="couldnt_open_app">Impossible d\'ouvrir l\'application</string>
+    <string name="network_error">Erreur réseau: %1$s</string>
+    <string name="app_verification_failed">Échec de la vérification de l\'application: %1$s</string>
+    <string name="failed_download_files">Échec du téléchargement des fichiers nécessaires: %1$s</string>
+    <string name="failed_download_repodata">Échec du téléchargement des données du dépôt: %1$s</string>
+    <string name="failed_verify_repodata">Échec de la vérification des données du dépôt: %1$s</string>
+    <string name="failed_decode_repodata">Échec du décodage des données du dépôt: %1$s</string>
+    <string name="error_parsing_files">Erreur lors de l\'analyse des fichiers de l\'application: %1$s</string>
+    <string name="app_doesnt_support_device">Application incompatible avec votre appareil: %1$s</string>
+    <string name="unknown_host_error">Erreur d\'hôte inconnue: %1$s</string>
+    <string name="user_restriction">Erreur de restriction d\'utilisateur: %1$s</string>
+    <string name="version">Version: %1$s</string>
+    <string name="version_code">Code de version: %1$d</string>
+    <string name="device_language_unsupported">la langue de votre appareil n\'est pas prise en charge</string>
+    <string name="device_density_unsupported">la densité d\'affichage de votre appareil n\'est pas prise en charge</string>
+    <string name="device_abi_unsupported">l\'architecture de votre appareil n\'est pas prise en charge</string>
+    <string name="app_signer_mismatch">l\'application n\'est pas signée par le(s) signataire(s) requis.\nAttendu: %1$s</string>
+    <string name="no_app_signers">aucun signataire d\'application trouvé.\nAttendu: %1$s</string>
+    <string name="version_downgrade">la version %1$d est inférieure à la version %2$d minimale</string>
+    <string name="app_id_mismatch">l\'ID de l\'application %1$s ne correspond pas à l\'ID %2$s attendu</string>
+    <string name="version_mismatch">version %1$s attendue, la version reçue est %2$s</string>
+    <string name="version_code_mismatch">le code de version attendu est %1$d, le code de version reçu est %2$s</string>
```

Comment:
The `$s` in `version_code_mismatch` needs to be a `$d` for the build to pass.

Reply:
It's changed, thanks for your help